compliance/compliance-server/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use clap::Parser;
+use compliance_api::{ComplianceCheckResponse, ComplianceStatus};
+use error::ComplianceServerError;
+use renegade_util::telemetry::{setup_system_logger, LevelFilter};
+use warp::{reply::Json, Filter};
+
+pub mod error;
+
+/// The CLI for the compliance server
+#[derive(Debug, Clone, Parser)]
+#[command(about = "The CLI for the compliance server")]
+struct Cli {
+    /// The port to listen on
+    #[arg(short, long)]
+    port: u16,
+    /// The Chainalysis API key
+    #[arg(long)]
+    chainalysis_api_key: String,
+}
+
+#[tokio::main]
+async fn main() {
+    setup_system_logger(LevelFilter::INFO);
+    let cli = Cli::parse();
+
+    // Get compliance information for a wallet
+    let chainalysis_key = cli.chainalysis_api_key.clone();
+    let compliance_check = warp::get()
```

Comment:
just curious, why did we choose to use `warp` here instead of `hyper` (as we do in the relayer)?

Reply:
we don't need the low level interface hyper gives us here. In the relayer it's helpful for more custom auth flows and routing, but here we just want something simple -- which warp gives us